connect/eaas/core/decorators.py

```diff
@@ -121,5 +122,25 @@ def wrapper(cls):
     return wrapper
 
 
+def transformation(name, description, edit_dialog_ui):
+    def wrapper(cls):
+        if hasattr(cls, TRANSFORMATION_ATTR_NAME):
```

Comment:
I think this if is not needed. If people put the same decorator twice is not our problem.